test/e2e/suites/e2e/e2e_test.go

```diff
@@ -229,6 +230,12 @@ var _ = Describe("e2e tests", func() {
 
 			os.Setenv("CLUSTER_EXTRA_NET_1", extraNet1.ID)
 			os.Setenv("CLUSTER_EXTRA_NET_2", extraNet2.ID)
+
+			policyName := "demo-policy"
+			policyID, err = shared.GetOpenStackQosPolicyID(e2eCtx, policyName)
+			Expect(err).To(BeNil())
+			Expect(policyID).NotTo(BeNil(), "Neutron QoS policy does not exist: %s", policyName)
+			os.Setenv("CLUSTER_QOS_POLICY_ID", policyID)
```

Comment:
I'm missing the bits later in this test to confirm that the port has been created with a specific Policy ID.
@mdbooth I might use your help on this one